### PR TITLE
New prop: showOrderTitle

### PR DESCRIPTION
**Pull request by khrizzcristian**
#### What problem is this solving?
This prop allows users to hide the selected option label. ("Relevance", "Name", etc..)

### CODE REVIEW

**Comment by klzns** [comment]
Hi @khrizzcristian thanks for the pull request! Can you provide a link to a workspace with using that feature?

**Comment by khrizzcristian** [comment]
> Hi @khrizzcristian thanks for the pull request! Can you provide a link to a workspace using that feature?

Hello. I have no active links. I tested on https://longo--distinctive.myvtex.com/ but being the live application it didn't read the prop I was giving from the category page

**Comment by khrizzcristian** [comment]
Hello. I just update the pull request
Here is a link where i test it > https://longo--distinctive.myvtex.com/barbati

I just added a new css handler for filter navigator "filterIsOpen" which allows us to add css on the title area when the filter is opened. In my case, it's just a white background

File: react/components/SelectionListOrderBy.js [hunk 1]
```diff
@@ -105,7 +106,7 @@ const SelectionListOrderBy = ({
           >
             {sortByMessage}
           </span>{' '}
-          {getOptionTitle(orderBy)}
+          {showOrderTitle === true ? getOptionTitle(orderBy) : ''}
```
Comment (icazevedo) [inline react/components/SelectionListOrderBy.js:109]:
```suggestion
          {showOrderTitle ? getOptionTitle(orderBy) : null}
```

**Review by icazevedo — CHANGES_REQUESTED** [review]

File: react/components/SelectionListOrderBy.js (hunk 1 above)
Comment (khrizzcristian) [reply react/components/SelectionListOrderBy.js]: Done :) I tried in that way in the first place and it didn't work.. that's why i put `=== true`

File: docs/README.md [hunk 2]
```diff
@@ -297,7 +297,8 @@ Check out the [**Product Summary documentation**](https://vtex.io/docs/component
 
 | Prop name       | Type            | Description                                                                                                  | Default value |
 | --------------- | --------------- | ------------------------------------------------------------------------------------------------------------ | ------------- |
-| `hiddenOptions` | `[string]` | Indicates which sorting options will be hidden. (e.g. `["OrderByNameASC", "OrderByNameDESC"]`) | `undefined`       |
+| `hiddenOptions` | `[string]` | Indicates which sorting options will be hidden. (e.g. `["OrderByNameASC", "OrderByNameDESC"]`) | `undefined`      |
+| `showOrderTitle` | `[boolean]` | Show or hide the selected order value (e.g. `Relevance`)                                     | `true`           |
```
Comment (guerreirobeatriz) [inline docs/README.md:301]:
```suggestion
| `hiddenOptions` | `[string]` | Indicates which sorting options will be hidden. (e.g. `["OrderByNameASC", "OrderByNameDESC"]`) | `undefined`      |
| `showOrderTitle` | `boolean` | Whether the selected order value (e.g. `Relevance`) will be displayed (`true`) or not (`false`).  | `true`           |
```

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]

**Review by guerreirobeatriz — APPROVED** [review]

**Comment by hiagolcm** [comment]
Is the  https://longo--distinctive.myvtex.com/barbati missing the `showOrderTitle` change? I can't find it in the code.

image

**Comment by khrizzcristian** [comment]
> Is the https://longo--distinctive.myvtex.com/barbati missing the `showOrderTitle` change? I can't find it in the code.
> 
> image

Hello. Yes. I unlinked the app. If you want we can test together live

**Comment by khrizzcristian** [comment]
Hello. Can anyone else examine my PR?
I have clients waiting for these changes.
Thank you.

**Comment by hiagolcm** [comment]
@khrizzcristian Why did you unlink? We need a workspace to test it.
Could you create another one? please?

**Comment by khrizzcristian** [comment]
> @khrizzcristian Why did you unlink? We need a workspace to test it.
> Could you create another one?

Hello. Of course. I'll comment here when it's done.

**Review by klzns — CHANGES_REQUESTED** [review]

**Comment by khrizzcristian** [comment]
> @khrizzcristian Why did you unlink? We need a workspace to test it.
> Could you create another one?

Hello. Here is the workspace 
https://searchresult--distinctive.myvtex.com/Barbati

**Review by klzns — APPROVED** [review]

**Review by hiagolcm — APPROVED** [review]

**Review by icazevedo — APPROVED** [review]

**Pull request merged**